manifests/init.pp

```diff
@@ -754,4 +755,9 @@
     validate_hash($keys_real)
     create_resources('ssh_authorized_key', $keys_real)
   }
+
+  if $sshd_addressfamily != undef {
+    validate_re($sshd_addressfamily, '^(any|ipv4|ipv6|inet)$',
```

Comment:
`any`, `inet` and `inet6` are the supported values 

http://www.openbsd.org/cgi-bin/man.cgi/OpenBSD-current/man5/sshd_config.5?query=sshd_config